tests/python/pytorch/graphbolt/impl/test_c_sampled_subgraph.py

```diff
@@ -0,0 +1,93 @@
+import multiprocessing as mp
+import os
+import tempfile
+import time
+import unittest
+
+import backend as F
+
+import dgl
+import dgl.graphbolt as gb
+import gb_test_utils as gbt
+
+import pytest
+import torch
+
+
+def subprocess_entry(q):
+    num_nodes = 5
+    num_edges = 12
+    indptr = torch.LongTensor([0, 3, 5, 7, 9, 12])
+    indices = torch.LongTensor([0, 1, 4, 2, 3, 0, 1, 1, 2, 0, 3, 4])
+    type_per_edge = torch.LongTensor([0, 0, 1, 0, 1, 0, 1, 0, 1, 0, 0, 1])
+    assert indptr[-1] == num_edges
+    assert indptr[-1] == len(indices)
+    ntypes = {"n1": 0, "n2": 1, "n3": 2}
+    etypes = {("n1", "e1", "n2"): 0, ("n1", "e2", "n3"): 1}
+    metadata = gb.GraphMetadata(ntypes, etypes)
+
+    # Construct CSCSamplingGraph.
+    graph = gb.from_csc(
+        indptr, indices, type_per_edge=type_per_edge, metadata=metadata
+    )
+
+    adjs = []
+    seeds = torch.arange(5)
+
+    # Sampling.
+    for hop in range(2):
+        sg = graph.sample_neighbors(seeds, torch.LongTensor([2]))
+        seeds = sg.indices
+        adjs.append(sg)
+
+    # 1. Put.
```

Comment:
The 1, 2, 3, 4 which representing the order of the operators is not very clear, and also how do you guarantee the #2 happens after #1?